doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc

```diff
@@ -1,13 +1,24 @@
 [id='test-designer-run-test-proc']
-= Running a test scenario
+= Running the test scenarios
 
 After creating a test scenario template and defining the test scenarios, you can run the tests to validate your business rules and data.
 
 .Procedure
-. At the top of the test scenarios designer, click *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon.
-. The *Test Report* panel pops out from the right of the designer and displays the test result overview and the scenario status.
-. After the tests are executed, click *View Alerts* to open the *Alerts* panel at the bottom of the designer for the error messages.
-. In case the tests fails, review the error messages. Messages with a red cross indicate test failures where as those with a green check mark indicate successful tests.
+. To run the defined test scenarios, choose one of the following methods:
```

Comment:
I think we can change this to more closely follow the nonsequential actions section of the IBMSG p88-89.

. To run defined test scenarios, do one of the following tasks:
* To execute all the available test scenarios in your project inside multiple assets, in the upper-right corner of your project page, click *Test*.
* To execute all the available test scenarios defined in a `.scesim` file, at the top of the Test Scenario designer, click *Run Test*.
* To run a single test scenario defined in a `.scesim` file, right-click the row of the test scenario you want to run and select *Run scenario*.

Reply:
Ack. 
I will try to incorporate this. 
Thank you for the IBMSG reference. :)